src/network.lisp

```diff
@@ -34,16 +41,21 @@
                                       and
                                         collect value))
            (response
-             (dexador:post url
+             (if *proxyp*
```

Comment:
You don't need addional flag, just check if proxy not nil.

Reply:
Thanks for the modification, I'm a newbie with Common Lisp.